File: app/src/main/java/com/utynote/components/search/data/geojson/entities/SearchData.java

package com.utynote.components.search.data.geojson.entities;

import java.util.List;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class SearchData {

    @SerializedName("geocoding")
    @Expose
    public Geocoding geocoding;
    @SerializedName("type")
    @Expose
    public String type;
    @SerializedName("features")
    @Expose
    public List<Feature> features = null;
    @SerializedName("bbox")
    @Expose
    public List<Double> bbox = null;

}

File: app/src/main/java/com/utynote/components/search/data/geojson/entities/SearchData.kt
package com.utynote.components.search.data.geojson.entities

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class SearchData {

    @SerializedName("geocoding")
    @Expose
    var geocoding: Geocoding? = null
    @SerializedName("type")
    @Expose
    var type: String? = null
    @SerializedName("features")
    @Expose
    var features: List<Feature>? = null
    @SerializedName("bbox")
    @Expose
    var bbox: List<Double>? = null

}
